block/manager.go

```diff
@@ -802,7 +801,7 @@ func (m *Manager) publishBlock(ctx context.Context) error {
 		if err != nil {
```

Comment:
comment on line 810: `here we set the SignedHeader.DataHash, and SignedHeader.Commit as a hack` can be updated to say `Signature` instead.

Reply:
On a similar note, `block_spec.md` can be updated here:

`Assert that len(SignedHeader.Commit.Signatures) == 1 (Exactly one signer check)`